components/script/dom/htmllinkelement.rs

```diff
@@ -1387,10 +1136,10 @@ impl FetchResponseListener for PreloadContext {
     }
 }
 
-impl ResourceTimingListener for PreloadContext {
+impl ResourceTimingListener for LinkFetchContext {
     fn resource_timing_information(&self) -> (InitiatorType, ServoUrl) {
         (
-            InitiatorType::LocalName(self.url.clone().into_string()),
+            InitiatorType::LocalName(self.type_.clone()),
```

Comment:
All of these test changes are because of this change. We were reporting the wrong type information in performance timing and all tests use performance machinery to observe requests.